tests/capping/test_BaseCappingTransformer.py

```diff
@@ -431,10 +432,6 @@ def test_non_cap_column_left_untouched(
         if _check_if_skip_test(transformer, df, lazy=False, from_json=from_json):
             return
 
-        # if transformer is not polars compatible, skip polars test
```

Comment:
Assuming this has been removed since BaseCappingTransformer is polars compatible now?